lib/phoenix_live_view/channel.ex

```diff
@@ -235,6 +240,31 @@ defmodule Phoenix.LiveView.Channel do
     {:noreply, new_state}
   end
 
+  def handle_info({@prefix, :report_writer_error, channel_pid, reason}, state) do
+    case Map.fetch(state.upload_pids, channel_pid) do
+      {:ok, {ref, entry_ref, cid}} ->
```

Comment:
Nitpick. Prefer pattern matching to Map.fetch:

```
case state.upload_pids do
  %{^channel_pid => {ref, entry_ref, cid}} ->
```

Reply:
I saw this pattern indeed around the code base and I was just wondering: is this because of style or performance? or both? :)

Reply:
Mostly style. It is a very very very tiny tiny bit faster too as it does not allocate a tuple.